src/pretix/control/forms/event.py

```diff
@@ -240,6 +240,21 @@ class EventSettingsForm(SettingsForm):
         help_text=_("Require customers to fill in the names of all attendees."),
         required=False
     )
+    attendee_emails_asked = forms.BooleanField(
+        label=_("Ask for attendee e-mails"),
+        help_text=_("Ask for an e-mail address for all tickets which include admission to the event. Important: For "
```

Comment:
Hmmm, I get your point and this should totally be improved but I believe your suggestion is even more unclear. It's hard to get the distinction between **purchaser** emails and **attendee** emails across and I think "Attendee emails addresses are required regardless of this setting for confirmation links" makes it even more confusing.